subcommands/targets/offline-update.go

```diff
@@ -65,25 +71,54 @@ func init() {
 		"Skip fetching Target Apps")
 	offlineUpdateCmd.Flags().BoolVarP(&ouAllowMultipleTargets, "allow-multiple-targets", "", false,
 		"Allow multiple targets to be stored in the same <dst> directory")
+	offlineUpdateCmd.Flags().StringVarP(&ouWave, "wave", "", "",
+		"Name of the wave Target content of which to fetch")
```

Comment:
>I would also keep this argument definition next to where we define the --prod flag (near line 64-65).

What's the difference?

Reply:
A difference is that similar options are grouped closer to each other.

Reply:
@kprosise Please, suggest the better wording for `"Name of the wave Target content of which to fetch"` -  this is a description of the CLI parameter `wave`, a value of this parameter should be a wave name. A user specifies a wave name, then the command gets the wave targets and find the specified target among them.

Reply:
I just moved `wave` param to the tag/prod group. 0665b26

Reply:
would a simplified  "Name of wave Target content to fetch" work?

Reply:
@mike-sul @kprosise when I read your sentences I am under impression that this argument provides a `name of a (wave) Target`, but it provides `a name of a Wave`. We need to somehow separate these two nouns, so that confusion is removed.

Reply:
The other aspect to take into consideration is that we have a similar parameter `--prod` with [this description](https://github.com/foundriesio/fioctl/commit/0665b2626a840f0cdb24a990347307e6d6fc615d#diff-54ddea03a308be581ec1bc0426e2f90dcc0bdb21073eaa68ae351e32217b53ebL65):
```
      --prod                     Instruct to fetch content of production Target
```

Ideally, when we add a new parameter with a similar meaning, it needs to have a similar look and feel.
Namely, I mean that the updated help would ideally look like this:

```
      --prod                     Instruct to fetch content of production Target
      --wave string              Something with a similar look and feel as the above
```

Reply:
Maybe as simple as `Instruct to fetch content of wave Target`?

Reply:
@vkhoroz @kprosise What do think about the suggested above param descriptions?

Reply:
@mike-sul I like the second one with the semicolon.

Reply:
Updated via force push.